### PR TITLE
Order fix

### PR DESCRIPTION
**Pull request by tikitko**

### CODE REVIEW

File: blog-server-services/src/impls/rbatis_author_service.rs [hunk 1]
```diff
@@ -20,15 +20,15 @@ impl_select!(Author {select_by_telegram_id(telegram_id: &u64) -> Option =>
 impl_select!(Author {select_by_slug(slug: &String) -> Option => 
     "`WHERE slug = #{slug} LIMIT 1`"});
 impl_select!(Author {select_all_with_offset_and_limit(offset: &u64, limit: &u64) => 
-    "`LIMIT #{limit} OFFSET #{offset}`"});
+    "`ORDER BY author.editor DESC, author.blocked ASC, author.registered_at ASC LIMIT #{limit} OFFSET #{offset}`"});
```
Comment (YoshkiMatryoshki) [inline blog-server-services/src/impls/rbatis_author_service.rs:23]: do we even need to display blocked users?
could we just filter this records out?

File: blog-server-services/src/impls/rbatis_author_service.rs [hunk 2]
```diff
@@ -20,15 +20,15 @@ impl_select!(Author {select_by_telegram_id(telegram_id: &u64) -> Option =>
...
 impl_select!(Author {select_all_by_query_with_offset_and_limit(query: &String, offset: &u64, limit: &u64) => 
-    "`WHERE author.slug ILIKE '%' || #{query} || '%' OR author.first_name ILIKE '%' || #{query} || '%' OR author.middle_name ILIKE '%' || #{query} || '%' OR author.last_name ILIKE '%' || #{query} || '%' LIMIT #{limit} OFFSET #{offset}`"});
+    "`WHERE author.slug ILIKE '%' || #{query} || '%' OR author.first_name ILIKE '%' || #{query} || '%' OR author.middle_name ILIKE '%' || #{query} || '%' OR author.last_name ILIKE '%' || #{query} || '%' ORDER BY author.editor DESC, author.blocked ASC, author.registered_at ASC LIMIT #{limit} OFFSET #{offset}`"});
 
 impl Author {
     #[py_sql(
         "
         SELECT COUNT(1) \
-        FROM author \
+        FROM author
```
Comment (YoshkiMatryoshki) [inline blog-server-services/src/impls/rbatis_author_service.rs:31]: this will work fine?

Comment (tikitko) [reply blog-server-services/src/impls/rbatis_author_service.rs:31]: looks yes...

**Review by YoshkiMatryoshki — APPROVED** [review]

File: blog-server-services/src/impls/rbatis_author_service.rs (hunk 1 above)
Comment (tikitko) [reply blog-server-services/src/impls/rbatis_author_service.rs:23]: yep, let's keep at the end

File: blog-server-services/src/impls/rbatis_comment_service.rs [hunk 3]
```diff
@@ -7,7 +7,7 @@ pub fn create_rbatis_comment_service(rb: RBatis) -> Box<dyn CommentService> {
 }
 
 impl_select!(Comment {select_all_by_post_id_with_limit_and_offset(post_id: &u64, limit: &u64, offset: &u64) => 
-    "`WHERE post_id = #{post_id} LIMIT #{limit} OFFSET #{offset}`"}, "post_comment");
+    "`WHERE post_id = #{post_id} ORDER BY created_at ASC LIMIT #{limit} OFFSET #{offset}`"}, "post_comment");
```
[the remark this answers is not in the document]
Comment (tikitko) [reply blog-server-services/src/impls/rbatis_comment_service.rs:10]: add new comment field at the end, and new comments should appear near this field.

**Pull request merged**